app/views/admin/enrollment_exemptions/show/_correspondant.html.erb

```diff
@@ -1,51 +1,37 @@
-<div class="row">
-  <div class="col-md-12">
-    <div class="panel panel-default">
-      <div class="panel-heading panel-title"><%= glyphicon_tag :user, text: t('.title') %></div>
-      <div class="panel-body" id="correspondence-contact-details">
-        <% contact = presenter.correspondence_contact %>
-
-        <% if contact %>
-          <strong><%= contact.full_name %></strong>
-          <% if contact.position? %>
-            <br/>
-            <%= t('.position') %> <%= contact.position %>
-          <% end %>
-          <br/>
-          <% if contact.email_address? %>
-            <br/>
-            <%= mail_to contact.email_address %>
-          <% end %>
-          <% if contact.telephone_number? %>
-            <br/>
-            <%= link_to(contact.telephone_number, "tel:#{contact.telephone_number.gsub(/[^0-9]/, '')}") %>
-          <% end %>
-        <% end %>
-      </div>
-    </div>
-  </div>
+<div id="correspondence-contact-details" class="govuk-!-padding-bottom-2">
+  <h2 class="govuk-heading-m"><%= t ".contact_header" %></h2>
+
+  <% contact = presenter.correspondence_contact %>
+  <p><%= contact.full_name %></p>
+
+  <% if !contact.position? %>
```

Comment:
Should this be `contact.position?` instead? Looks like it's changed but not sure why

Reply:
Well spotted! My debugging code.

Reply:
Fixed in: https://github.com/DEFRA/flood-risk-back-office/pull/312/commits/1a9e427fbf00e009ae85d666259fbaa72c0b0e13